tests/test_obs_test.py

```diff
@@ -0,0 +1,70 @@
+# See the COPYRIGHT and LICENSE files in the top-level directory of this package
+# package for notices and licensing terms.
+import os
+import sys
+import unittest
+
+import lsst.utils.tests
+from lsst.utils import getPackageDir
+from lsst.afw.geom import Extent2I, Point2D
+import lsst.obs.base.tests
+from lsst.obs.base.tests import make_bbox
+import lsst.obs.test
+
+
+class TestObsTest(lsst.obs.base.tests.ObsTests, lsst.utils.tests.TestCase):
+    def setUp(self):
+        product_dir = getPackageDir('obs_test')
+        self.data_dir = os.path.join(product_dir, 'data', 'input')
+
+        super(TestObsTest, self).setUp()
+
+        self.butler = lsst.daf.persistence.Butler(root=self.data_dir)
+        self.mapper = lsst.obs.test.TestMapper(root=self.data_dir)
```

Comment:
This is breaking encapsulation.  `TestObsTest` should only set things that it cares about; it should not have to set up things for its parent class.  If something needs to be set up for the parent class, then you should call a method in the parent class to do that.  I should be able to change the names of instance variables in the parent class without this subclass knowing anything about it (I know that's not always possible, but it's impossible to such a large extent here that I think it's a bad idea).